spec/models/manageiq/providers/redhat/infra_manager/refresh/refresher_4_1_spec.rb

```diff
@@ -206,8 +206,8 @@ def assert_specific_host
       :cpu_type             => "Intel(R) Xeon(R) CPU           E5504  @ 2.00GHz",
       :manufacturer         => "",
       :model                => "",
-      :number_of_nics       => nil,
-      :memory_mb            => 56333,
+      :number_of_nics       => 2,
+      :memory_mb            => 7806,
```

Comment:
@agrare the bug is that the nics were not parsed although the info was in the responce. So the vcr did not change but the nics are now parsed.